src/iperf_pthread.c

```diff
@@ -6,19 +6,10 @@
  * as Android NDK does not support `pthread_cancel()`.
  */
 
+#include <string.h>
```

Comment:
error: call to undeclared library function 'memset' with type 'void *(void *, int, unsigned long)'; ISO C99 and later do not support implicit function declarations [-Wimplicit-function-declaration]